Makefile

```diff
@@ -90,7 +90,11 @@ test/unit:
 test/e2e: export GH_CLIENT_ID := 1234
 test/e2e: export GH_CLIENT_SECRET := 1234
 test/e2e: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/configmaps cluster/prepare/crd deploy/integreatly-installation-cr.yml
-	INTEGREATLY_OPERATOR_DISABLE_ELECTION=true operator-sdk --verbose test local ./test/e2e --namespace $(NAMESPACE) --up-local --go-test-flags "-timeout=60m" --debug
+ifdef OPENSHIFT_BUILD_NAMESPACE
```

Comment:
I think what might be better here is to actually set an additional env var on the job in the ci-operator config, e.g. here: https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L32

This would allow us locally to specify what image to use also.